torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10596,6 +10596,9 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         skips=(
             DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_noncontiguous_samples', device_type='cpu'),
             DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit', dtypes=(torch.float32,)),
+            DecorateInfo(toleranceOverride({torch.float16: tol(atol=0.05, rtol=0.003),
```

Comment:
Are the current failures only on ROCm, as @kit1980 suggests? If so maybe these tolerances should only be applied for ROCm?

Reply:
> The failing test is "test_nvfuser_correctness". Are we running nvfuser on Rocm GPU? Does we expected this to work? @kit1980 Or shall we `@skipIfRocm` for this test?

I see now that `test_nvfuser_correctness__softmax_backward_data_cuda_bfloat16` and `test_nvfuser_correctness__softmax_backward_data_cuda_float16` were introduced by https://github.com/pytorch/pytorch/pull/8876 and were never run before that.

So yes, we should just disable the tests for ROCm.